e2e/pages/disk-sidebar.po.ts

```diff
@@ -16,4 +21,88 @@ export class DiskSidebar extends CloudstackUiPage {
   clickClose() {
     element(by.css('.backdrop.ng-star-inserted')).click();
   }
+
+  setDescription(description) {
+    const EC = browser.ExpectedConditions;
+    const elem = element(by.css('.mat-card-content span'));
+    browser.wait(EC.elementToBeClickable(elem), 2000, ' Description is not clickable');
+    elem.click();
+
+    browser.wait(
+      EC.presenceOf(element(by.name('textField'))),
+      5000,
+      "Text area of disk description doesn't present",
+    );
+    element(by.name('textField')).clear();
+    element(by.name('textField')).sendKeys(description);
+    element
+      .all(by.css('.buttons button'))
+      .last()
+      .click();
+    element.all(by.css('.buttons button')).isPresent();
```

Comment:
Why do you need the last line of code in this function? Maybe it is superfluous?

Reply:
Here I check the button "Cancel", but I incorrectly pointed to it